runserver

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+
+#download all models in environment variable NLP_SERVICE_MODELS_JSON
+echo "downloading models"
+python download.py
+
+#run the server
+echo "starting server"
+gunicorn -b 0.0.0.0:5000 --worker-connections 1000 app:app
```

Comment:
Sure, let's use `gunicorn`. But we need to configure it better:
- the process count -- https://docs.gunicorn.org/en/stable/settings.html#worker-processes
- the worker type -- `gthread` because [it works better with long running tasks](https://docs.gunicorn.org/en/stable/design.html#how-many-threads)
- `--worker-connections` - we're not using `gevent` or `eventlet`, so remove this
- `--timeout` - set this to 2X what it takes to run your lib on some large model on the 4.5M King James Bible 
- `--keep-alive=120`


We want some new envs, called:
- `GUNICORN_WORKERS` - default 2
- `GUNICORN_THREADS` - default 30


The default values you can set in the Dockerfile with `ENV GUNICORN_WORKERS=2` lines. These defaults should work properly on the 40+ workers we usually boot up on large servers.